ibmcloudant/features/changes_follower.py

```diff
@@ -178,11 +180,10 @@ def _request_callback(self):
                 self._buffer.join()
                 if self._stop.is_set():
                     raise StopIteration
-                self._buffer.put(
-                    [ChangesResultItem.from_dict(item) for item in results]
-                )
+                self._buffer.put(results)
             except Exception as e:
-                self.logger.debug(f'Exception getting changes {e}')
+                if type(e) is not StopIteration:
```

Comment:
That deDRY thing though, because the difference between `StopIteration` and terminal errors is only a log message. But sure, done in https://github.com/IBM/cloudant-python-sdk/pull/731/commits/281f0b94866b45b543a3710495d4bf3979380c6d

Reply:
Yeah I thought about the deDRY, but we were already repeating the buffer join/put once anyway and we were repeating the exception type check, so we were going to end up with something repeating either way.